extensions/notebook/src/book/bookModel.ts

```diff
@@ -153,8 +155,8 @@ export class BookModel implements azdata.nb.NavigationProvider {
 							let uriToNotebook: vscode.Uri = vscode.Uri.file(pathToNotebook);
 							if (!this._allNotebooks.get(uriToNotebook.fsPath)) {
 								this._allNotebooks.set(uriToNotebook.fsPath, notebook);
-								notebooks.push(notebook);
 							}
+							notebooks.push(notebook);
```

Comment:
Was this a bug? When would you expect this to happen - shouldn't there only be one notebook per fsPath anyways?

Reply:
I introduced this when making changes for CreateBook so kind of a regression that I fixed.

---
In reply to: [388545961](https://github.com/microsoft/azuredatastudio/pull/9453#discussion_r388545961) [](ancestors = 388545961)